jsonParser/CdlInf.py

```diff
@@ -0,0 +1,126 @@
+from sys import flags
+from Target import Target
+import json
+
+@Target.register
+class CdlInf(Target):
+
+    def __control_quorum(self, text, quorum) -> bool:
+        if not quorum:
+            return False
+        result = True
+        if not self.word_not_in_control("NO QUORUM", text):
+            result = False
+            text[self.i] = text[self.i].replace("NO QUORUM", " ").strip()
+        return result
+
+    def __find_name_department(self, text, quorum) -> str:
+        self.word_not_in_update("ELEZIONI", text)
+        self.i += 1
+        quorum[0] = self.__control_quorum(text, quorum[0])
+        nome = text[self.i]
+        self.i += 1
+        while self.word_not_in_control("BIENNIO", text):
+            if len(text[self.i]) > 5:
+                quorum[0] = self.__control_quorum(text, quorum[0])
+                nome += "\n" + text[self.i]
+            self.i += 1
+        return nome.strip()
+    
+    def __seggi_da_assegnare(self, text) -> int:
+        split_text = text[self.i].split()
+        try:
+            return int(split_text[len(split_text)-1])
+        except ValueError:
+            self.i += 1
+            split_text = text[self.i].split()
+            return int(split_text[len(split_text)-1])
+
+    def __get_candidati(self, text, eletti, non_eletti, current_quorum) -> bool:
+        self.word_not_in_update("CANDIDATI", text)
+        self.i += 1
+        quorum = current_quorum
+        while self.word_not_in_control("SCHEDE", text):
+            split_text = text[self.i].split()
+            if self.is_integer(split_text[0]):
+                split_text.pop(0)
+            if len(split_text) > 0:
+                nome_candidato = ""
+                voti_candidato = 0
+                eletto = False
+                for s in split_text:
+                    if self.is_integer(s):
+                        voti_candidato = s
+                    elif "ELETTO" == s.upper():
+                        eletto = True
+                    else:
+                        nome_candidato += (s + " ")
+                if not self.word_not_in_control("NO QUORUM", text):
+                    quorum = False
+                    nome_candidato = nome_candidato.replace("NO QUORUM", " ").strip()
+                if not self.word_not_in_control("EX AEQUO", text):
+                    nome_candidato = nome_candidato.replace("EX AEQUO", " ").strip()
+                if eletto:
+                    eletti.append({
+                        "nome_candidato": nome_candidato.strip(),
+                        "voti": int(voti_candidato)
+                    })
+                else:
+                    non_eletti.append({
+                        "nome_candidato": nome_candidato.strip(),
+                        "voti": int(voti_candidato)
+                    })
```

Comment:
If you want you could write
```suggestion
                e = {
                        "nome_candidato": nome_candidato.strip(),
                        "voti": int(voti_candidato)
                };
                if eletto:
                    eletti.append(e)
                else:
                    non_eletti.append(e)
```